python/ray/air/_internal/usage.py

```diff
@@ -56,6 +58,45 @@
     "ResourceChangingScheduler",
 }
 
+AIR_ENV_VARS = {
```

Comment:
We can also add [these Train env vars](https://github.com/ray-project/ray/blob/56418911557b47021a3e5af366f7d039af5c4a03/python/ray/train/constants.py#L52-L67)

Reply:
Good catch. I also found a few more that I missed: `COPY_DIRECTORY_CHECKPOINTS_INSTEAD_OF_MOVING_ENV`, `TRAIN_DISABLE_LAZY_CHECKPOINTING`